fastlane/Fastfile

```diff
@@ -648,15 +648,15 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
     match(
       type: "enterprise",
       team_id: get_required_env("INT_EXPORT_TEAM_ID"),
-      readonly: options[:readonly] || is_ci,
+      readonly: true,
```

Comment:
Playing captain obvious: this prevents us from regenerating these provisioning profiles from here.

I think this is good, the extra friction of having to edit the `Fastfile` might prevent unintended changes done while hitting keys in a rush, like when someone (me) does `git add .` instead of `git add -p`...

One suggestion I have is to extract this in a constant, so that if we'll  ever had the need to regenerate the profiles from here, it'll be easier to do it.

Having the value in a constant would also add a chance to document the rationale behind this value.

```ruby
# At the time of writing, the Enterprise Developer Account doesn't support logging in via
# API key. To avoid confusing errors when syncing then provisioning profiles locally because
# match can't access the Enterprise account, let's prevent it from trying to access it in the
# first place.
MATCH_READONLY = true
```

Reply:
> One suggestion I have is to extract this in a constant, so that if we'll ever had the need to regenerate the profiles from here, it'll be easier to do it.

My thought on this is actually that we should probably have a way of doing this outside the project. We have about one year before it affects us – I guess we'll see if we get to it by then? 😅